.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           activate-environment: sparse-dev
           allow-softlinks: true
           environment-file: ci/environment.yml
-          python-version: '3.10'
+          python-version: ${{ matrix.python }}
```

Comment:
Does it mean that `docs` stage will run against three different python versions? Is it beneficial? (I think that e.g. NumPy runs docs only against one python version)

Reply:
Hmm but `docs` job doesn't define `matrix` entry.

Reply:
Where is the `matrix.python` coming from here?